tiledb.rb

```diff
@@ -20,16 +19,19 @@ def install
 	cd "build" do
 	    args = %W[
 	      --prefix=#{prefix}
-              --enable-s3
+        --enable-s3
```

Comment:
It looks like the existing script is using a mix of tabs and spaces, so the indentation looks improper here. If you want to fix this, maybe a find-replace tabs to spaces.